src/Components/Lives/Lives.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components/macro';
+import { ReactComponent as Heart } from '../../Assets/heartGreen.svg';
+
+export default function Lives() {
+  return (
+    <LivesWrapper>
+      <Heart></Heart>
```

Comment:
Here you can use a void tag
```suggestion
      <Heart />
```